Makefile

```diff
@@ -18,10 +18,6 @@ down:
 build:
 	docker-compose build
 
-.PHONY: test
```

Comment:
Did you intend to remove this?

Reply:
It was a duplicate command. The same command is here:
https://github.com/I-GUIDE/catalogapi/blob/d0060c81b77bdf9928af387910d72a65b0e1f9d4/Makefile#L30